plugins/modules/system/open_iscsi.py

```diff
@@ -86,6 +86,12 @@
         - Whether the list of nodes in the persistent iSCSI database should be returned by the module.
         type: bool
         default: false
+    rescan:
+        description:
+        - rescan an established session for discovering new targets.
```

Comment:
```suggestion
        - Rescan an established session for discovering new targets.
        - When I(target) is omitted, will rescan all sessions.
```
I extracted that comment from the example; please check whether it is appropriate :)